reference/checkouts.v3.yml

```diff
@@ -1356,6 +1298,91 @@ definitions:
                       - amount
                       - sender
                       - recipient
+                custom_items:
```

Comment:
Is there a spec I can review to verify these properties? Just want to make sure that all of them apply to `custom_items` since they are different from the Storefront Checkouts API.